packages/scalars/src/scalars/EmailAddress.ts

```diff
@@ -0,0 +1,45 @@
+import {
+    GraphQLError,
+    GraphQLScalarType,
+    GraphQLScalarTypeConfig,
+    Kind,
+} from 'graphql';
+import { z } from 'zod';
+
+export const emailSchema = z.string().email();
+
+const emailValidation = (value: unknown): string => {
+    if (typeof value !== 'string') {
+        throw new GraphQLError(`Value is not string: ${JSON.stringify(value)}`);
+    }
+
+    const result = emailSchema.safeParse(value);
+
+    if (result.success) return result.data;
+    throw new GraphQLError(result.error.message);
+};
+
+export const GraphQLEmailAddressConfig: GraphQLScalarTypeConfig<
```

Comment:
I think we should not use specific names for the exports, it makes loading the scalars dynamically harder.

Each scalar could export:
- `schema` (currently `emailSchema`)
- `config` (currently `GraphQLEmailAddressConfig`)
- `scalar`(currently `GraphQLEmailAddress`)
- `typedef` (currently `EmailAddressTypeDefinition`)

Would be nice to define a `ScalarModule` type for this.